alembic.ini

```diff
@@ -60,7 +60,7 @@ version_path_separator = os  # Use os.pathsep. Default configuration used for ne
 # are written from script.py.mako
 # output_encoding = utf-8
 
-sqlalchemy.url = /paste/your/db/path
+sqlalchemy.url = sqlite:////home/akshaj/Documents/AIPlanet/GenAi/genai-stack/genai_stack/db
```

Comment:
Please dont commit these changes . Ill inform sunil as well that these things should be more dynamic .